dcp-0006/dcp-0006.mediawiki

```diff
@@ -0,0 +1,395 @@
+<pre>
+DCP: 0006
+Title: Decentralized Treasury
+Author: Marco Peereboom <marco@decred.org>
+Status: Defined
+Created: 2020-09-18
+License: CC0-1.0
+License-Code: ISC
+</pre>
+
+==Abstract==
+
+Spending from the Decred treasury is a manual process that is executed and
+governed by humans. The current process requires contractors to each produce a
+monthly invoice with a reimbursement report, denominated in USD, that is
+submitted to the project treasury auditors. If the invoice is approved a payout
+is made to the contractor-provided address from the treasury funds, using an
+averaged DCR/USD exchange rate over the month the invoice applies to. These
+payouts are made by Decred Holdings Group LLC ("DHG" for short), a
+conventional corporate entity which holds the treasury funds.
+
+This specifies modifications to the Decred treasury to make it a more
+decentralized process.
+
+The main changes are:
+
+* Move multisig UTXO based treasury address to a treasury account that can only be debited when Decred stakeholders agree to it.
+* Credit development subsidy to treasury account every block.
+* Add three new opcodes: <code>OP_TADD</code>, <code>OP_TSPEND</code>, and <code>OP_TGEN</code>.
+* The aforementioned opcodes are then in turn used to construct the three new types of stake transactions:
+*# Treasurybase transaction - treasury subsidy in every block.
+*# Treasury add transactions - method to credit the treasury account.
+*# Treasury spend transactions - method to debit the treasury account.
+
+==Motivation==
+
+At this present time, all Decred treasury payouts are handled by DHG. This is
+undesirable for various reasons, such as
+
+* Sovereignty over treasury spends is not vested in the stakeholders.
+* Risk of theft of funds.
+* Risk of key project members abandoning the project.
+* Risk of coercion being used against key project members.
+* Process of treasury spending is insufficiently transparent.
```

Comment:
I don't think tspend voting alone will make it more transparent. It is currently not transparent not because the payouts are handled by DHG, but because DHG is not posting financial statements. Transparency will be handled by upgrades to Politeia and CMS.

Reply:
Disagree. A TSpend will have linve items for everything paid.

Reply:
Do you refer to `TxOut[1:N]` ("individual payouts from the treasury account") in the Debit treasury section or something else?